### DISCUSSION TITLE
Included component tests

### DISCUSSION THREAD

**Discussion by GJSBRT** [shadcn-ui/ui#4619] [Ideas]
Just a thought. Would it be an idea to include a test for each component. As the idea of shadcn-ui is to change the components, I can see how tests could be very useful to see if a component is still behaving the same in terms of accessibility, functionality, etc.

I think this could be a nice addition. I am not too familiar with tests in javascript, so if this is a bad idea let me know :p.

**Comment by tobiasbueschel** [comment]
+1

However, this is a duplicate discussion as there is already one on the same topic here: https://github.com/shadcn-ui/ui/discussions/1248
